### DISCUSSION TITLE
[eme_if tag='#_OPT_IN' not working

### DISCUSSION THREAD

**Discussion by RealDeemer** [liedekef/events-made-easy#434] [Q&A] [answered]
Hi,

i want to display some content dynamically on the selected Value of #_OPT_IN. The line

[eme_if tag='#_OPT_IN'] or [eme_if tag='#_OPT_IN' eq=0]

is not working. Does anyone have some ideas how to get it work?

**Comment by liedekef** [comment, marked as answer]
eme_if is not evaluating changes made after a page is rendered.
If you want that: either look at "Dynamic data" to show an extra template based on a condition you set, or (probably better in your case): create some own jquery/javascript that shows/hide a specific span based on the value of another field. How to code custom jquery/javascript is outside the scope of EME support, but you can always take a peak at e.g. js/eme_admin_rsvp.js and see the function updateShowHideStuff() and how I use it in there.
